docs/resources/CHANNEL.md

```diff
@@ -215,7 +226,7 @@ The before and after keys are mutually exclusive, only one may be passed at a ti
 | after | snowflake | get messages after this message ID | false | absent |
 | limit | integer | max number of messages to return (1-100) | false | 50 |
 
-## Create Message % POST /channels/{channel.id#DOCS_CHANNEL/channel-object}/messages
+## Create Message % POST /channels/{channel.id#DOCS_CHANNEL/guild-channel-object}/messages
```

Comment:
This isn't just for guild channels. Would be nice to figure out a better way to display that.